package.json

```diff
@@ -49,6 +49,7 @@
   "devDependencies": {
     "@angular/compiler-cli": "8.2.0",
     "@ngtools/webpack": "8.2.0",
+    "@phenomnomnominal/angular-lazy-routes-fix": "^0.1.5",
```

Comment:
Why do we require this dep, is this expect to be used with this lazy loading syntax?

Reply:
Nope, I've left it by mistake. I was migrating the route configs with this TSLint rule.